geometry_msgs/Point max
geometry_msgs/Point min
geometry_msgs/Point center
geometry_msgs/Quaternion orientation